src/cmem/markdown/Markdown.tsx

```diff
@@ -36,7 +43,7 @@ export interface MarkdownProps extends TestableComponent {
      * Additional reHype plugins to execute.
      * @see https://github.com/remarkjs/react-markdown#architecture
      */
-    reHypePlugins?: PluggableList;
```

Comment:
Please add deprecation note that we do not support `PluggableList from "react-markdown/lib/react-markdown"` with the next major (v25), only the new unified one.